bin/dotd

```diff
@@ -129,7 +135,7 @@ end
 def should_i(question)
   loop do
     print "Should I #{question}? [#{underline 'Y'}es]/#{underline 'S'}kip/#{underline 'Q'}uit: "
-    response = gets.strip.downcase
+    response = flush_and_gets.strip.downcase
     puts ''
     if 'yes'.start_with?(response) # also catches blank/return ;)
```

Comment:
Instead of `flush_and_gets`, should the logic on this line be changed so that it doesn't catch blank/return responses?